### PR TITLE
Separate section for chain configuration in storage

### PR DESCRIPTION
**Pull request by alexvanin**

### CODE REVIEW

File: cmd/neofs-node/config/mainchain/config.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import (
+	"time"
+
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+)
+
+const (
+	subsection = "mainchain"
+
+	DialTimeoutDefault = 5 * time.Second
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/mainchain/config.go:12]:
```suggestion
	// DialTimeoutDefault is a default dial timeout of main chain client connection.
	DialTimeoutDefault = 5 * time.Second
```

File: cmd/neofs-node/config/mainchain/config.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+)
+
+// RPCEndpoint returns list of values of "rpc_endpoint" config parameter
+// from "morph" section.
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/mainchain/config.go:16]:
```suggestion
// from "mainchain" section.
```

File: cmd/neofs-node/config/mainchain/config.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+//
+// Returns empty list of not set.
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/mainchain/config.go:18]:
```suggestion
// Returns empty list if value is not a non-empty string array.
```

File: cmd/neofs-node/config/mainchain/config.go [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+func RPCEndpoint(c *config.Config) []string {
+	return config.StringSliceSafe(c.Sub(subsection), "rpc_endpoint")
+}
+
+// DialTimeout returns value of "dial_timeout" config parameter
+// from "mainchain" section.
+//
+// Returns DialTimeoutDefault if value is not set.
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/mainchain/config.go:26]:
```suggestion
// Returns DialTimeoutDefault if value is not a positive duration.
```

File: cmd/neofs-node/config/morph/config.go [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+		panic(fmt.Errorf("no morph chain notification endpoints, see `morph.notification_endpoint` section"))
+	}
+
+	return v
+}
+
+// DialTimeout returns value of "dial_timeout" config parameter
+// from "morph" section.
+//
+// Returns DialTimeoutDefault if value is not set.
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/morph/config.go:45]:
```suggestion
// Returns DialTimeoutDefault if value is not a positive duration.
```

File: cmd/neofs-node/config/mainchain/config.go (hunk 1 above)
Comment (alexvanin) [reply cmd/neofs-node/config/mainchain/config.go:13]: Fixed.

File: cmd/neofs-node/config/mainchain/config.go (hunk 2 above)
Comment (alexvanin) [reply cmd/neofs-node/config/mainchain/config.go]: Fixed.

File: cmd/neofs-node/config/mainchain/config.go (hunk 3 above)
Comment (alexvanin) [reply cmd/neofs-node/config/mainchain/config.go]: Fixed.

File: cmd/neofs-node/config/mainchain/config.go (hunk 4 above)
Comment (alexvanin) [reply cmd/neofs-node/config/mainchain/config.go]: Fixed.

File: cmd/neofs-node/config/morph/config.go (hunk 5 above)
Comment (alexvanin) [reply cmd/neofs-node/config/morph/config.go]: Fixed.

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**